medical_prescription_state_verify/__manifest__.py

```diff
@@ -1,21 +1,21 @@
 # -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
+# Copyright 2016-2017 LasLabs Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Medical Prescription Order State Verification',
```

Comment:
All single quotes in the file should be double. (e.g. 'category': 'Medical'  => "category": "Medical")